variables.tf

```diff
@@ -182,6 +182,12 @@ variable "iam_role_tags" {
 # Workspace SAML Configuration
 ################################################################################
 
+variable "create_saml" {
```

Comment:
Looks great - lets use the slightly longer form just to be explicit 
```suggestion
variable "create_saml_configuration" {
```

Reply:
Sounds good @bryantbiggs 👍